README.md

```diff
@@ -80,23 +80,23 @@ other transport related settings.
 To create a client to make API calls to Elasticsearch running on `http://localhost:9200`
 
 ```rust,no_run
-use elasticsearch::{Error, Elasticsearch};
+use elasticsearch::{Elasticsearch};
 
-fn run() {
-    let client = Elasticsearch::default();
+fn main() {
+    let _client = Elasticsearch::default();
```

Comment:
I realize client isn’t used, but would prefer to keep this as `client`, in line with all examples.